Formula/tarantool.rb

```diff
@@ -22,7 +22,8 @@ class Tarantool < Formula
   uses_from_macos "ncurses"
 
   def install
-    sdk = MacOS::CLT.installed? ? "" : MacOS.sdk_path
+    sdk = MacOS.sdk_path_if_needed
+    lib_suffix = MacOS.version <= ":catalina" ? "dylib" : "tbd"
```

Comment:
```suggestion
    lib_suffix = MacOS.version <= :catalina ? "dylib" : "tbd"
```

Reply:
Why would the lib suffix depend on the macOS version at all?

Reply:
It's a change Apple made in Big Sur. System libraries are no longer found in the file system. Instead, you have these `.tbd` files that contain symbol lists to avoid unnecessary `dlopen` calls